packages/components/molecules/f-media-element/src/FlexStyles.js

```diff
@@ -0,0 +1,117 @@
+import { MODIFIER_OPPOSITES_RULES_MAP, MODIFIER_RULES_MAP } from './constants';
+
+class FlexStyles {
+    flexDirection = '--row';
+
+    flexReverse = undefined;
+
+    defaultFlexDirection = '--row';
+
+    defaultFlexReverse = '';
+
+    operator = undefined;
+
+    operatorOpposite = undefined;
+
+    mediaSize = undefined;
+
+    /**
+     * Adds the specific part of css class string
+     * @param value
+     * @returns {string}
+     */
+    static returnReverseClassSegment (value) {
+        return value ? '--reverse' : '';
+    }
+
+    /**
+     * Adds the specific part of css class string
+     * @param value
+     * @returns {string}
+     */
+    static returnDirectionClassSegment (value) {
+        return value ? '--col' : '--row';
+    }
+
+    /**
+     * Function creates a string (css class name) using the previously set class properties
+     * @returns {*}
+     * @private
+     */
+    static createStyle (...values) {
```

Comment:
Would `createClassname` be a more accurate name?

Reply:
agreed will name `createClassName`